### PR TITLE
added ansible_delegated_host

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

magic vars

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_vars_facts.rst [hunk 1]
```diff
@@ -666,6 +666,8 @@ The batch size is defined by ``serial``, when not set it is equivalent to the wh
 
 ``ansible_check_mode`` is a boolean, set to ``True`` if you run Ansible with ``--check``.
 
+``ansible_delegatd_host`` is a sting, containing the ``inventory_hostname`` of the host you are delegating to in the task, only defined if doing delegation. ``inventory_hostname`` always reflects the original host the task targeted.
```
Comment (timway) [inline docs/docsite/rst/user_guide/playbooks_vars_facts.rst]: ... is a _string_, containing the ... #nit

**Review by timway — CHANGES_REQUESTED** [review]
I think it's just a simple spelling error.

File: docs/docsite/rst/user_guide/playbooks_vars_facts.rst (hunk 1 above)
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_vars_facts.rst]:
```suggestion
``ansible_delegated_host`` is a string, containing the ``inventory_hostname`` of the host you are delegating to in the task. This is only defined if you are doing delegation.  The ``inventory_hostname`` always reflects the original host the task targeted.
```

Comment (samccann) [reply docs/docsite/rst/user_guide/playbooks_vars_facts.rst]: something that confuses me here - I don't think it ever overwrites ``inventory_hostname``, right? If so then it should be ...is a string containing the hostname of the host you are delegating to ... (as in don't use inventory_hostname there).

**Comment by samccann** [comment]
Coming at this as a noob - ``ansible_delegated_hostname`` makes sense to me if the task is using ``delegated_to`` etc.

File: docs/docsite/rst/user_guide/playbooks_vars_facts.rst (hunk 1 above)
Comment (bcoca) [reply docs/docsite/rst/user_guide/playbooks_vars_facts.rst]: inventory_hostname is explicitly overwritten  in the delegated vars to the 'original host', if that is what you are asking

**Comment by bcoca** [comment]
yeah, i was just lazy and wanted to save myself a name, have not decided 100% on variable name yet and open to bikeshed

**Review by samccann — APPROVED** [review]
Docs portion LGTM

**Comment by bcoca** [comment]
rebuild_merge

**Pull request closed without merging**